msticpy/common/secret_settings.py

```diff
@@ -131,9 +131,10 @@ def __init__(self, tenant_id: str = None, use_keyring: bool = True):
 
         self.tenant_id = tenant_id or self._kv_settings.get("tenantid")
         if not self.tenant_id:
-            raise MPKeyVaultConfigException(
-                "TenantID must be specified in KeyVault settings section",
-                "in msticpyconfig.yaml",
+            raise MsticpyKeyVaultConfigError(
+                "Could not get TenantId from function parameters or configuration.",
+                "Please add this to the KeyVault section of msticpyconfg.yaml",
```

Comment:
msticpyconfg typo

Reply:
I think I found all of them - must have copied/pasted 10 times